common/src/sql/dbinit.sql

```diff
@@ -2160,6 +2160,260 @@ CREATE TABLE omicron.public.role_assignment (
      )
 );
 
+/*******************************************************************/
+
+/*
+ * External Networking
+ *
+ * **For more details on external networking see RFD 267**
+ */
+
+CREATE TYPE omicron.public.address_lot_kind AS ENUM (
+    'infra',
+    'pool'
+);
+
+CREATE TABLE omicron.public.address_lot (
+    id UUID PRIMARY KEY,
+    name STRING(63) NOT NULL,
+    description STRING(512) NOT NULL,
+    time_created TIMESTAMPTZ NOT NULL,
+    time_modified TIMESTAMPTZ NOT NULL,
+    time_deleted TIMESTAMPTZ,
+    kind omicron.public.address_lot_kind NOT NULL
+);
+
+CREATE UNIQUE INDEX ON omicron.public.address_lot (
+    name
+) WHERE
+    time_deleted IS NULL;
+
+CREATE TABLE omicron.public.address_lot_block (
+    id UUID PRIMARY KEY,
+    address_lot_id UUID NOT NULL,
+    first_address INET NOT NULL,
+    last_address INET NOT NULL
+);
+
+CREATE INDEX ON omicron.public.address_lot_block (
+    address_lot_id
+);
+
+CREATE TABLE omicron.public.address_lot_rsvd_block (
+    id UUID PRIMARY KEY,
+    address_lot_id UUID NOT NULL,
+    first_address INET NOT NULL,
+    last_address INET NOT NULL
+);
```

Comment:
I can see these not having a name or description... but shouldn't they have timestamp info?

Reply:
I could see that for `address_lot_block`. The reason it's not there for `address_lot_rsvd_block` is because that's an implementation detail of address allocation and was not intended to be visible from the outside.

Reply:
It may still be worth adding, even if it's only for our usage.